app/views/pre_registrations/new.html.erb

```diff
@@ -1,170 +1,162 @@
+                    <%= 'Personal Email' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.text_field :personalEmail ,:class => 'form-control', required: 'true',:maxlength=>"128"%>
+                </p>
+            </div>
+  
+            <div class="col-md-6">
+                <%= f.label :holiday_scheme_id, 'Holiday Scheme' do %>
+                    <%= 'Holiday Scheme' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :holiday_scheme_id, options_for_select(@holiday_scheme.collect { | holiday | [holiday.name,  holiday.id] }), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
+                </p>
+            </div>
+  
+            <div class="col-md-6">
+                <%= f.label :weekend_id, 'WeekEnd Scheme' do %>
+                    <%= 'WeekEnd Scheme' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :weekend_id, options_for_select(@weekend_scheme.collect { | weekend | [weekend.name,  weekend.id] }), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'}%>
                 </p>
             </div>
-    </div>
-
-    <%= f.submit "Submit " %>
-<% end %>
-
-
-<style>
-    table {
-        font-family: arial, sans-serif;
-        border-collapse: collapse;
-        width: 100%;
-    }
-
-    td, th {
-        border: 1px solid #dddddd;
-        text-align: left;
-        padding: 8px;
-    }
 
-    tr:nth-child(even) {
-        background-color: #ffffff;
-    }
-</style>
+        </div>
+  
+        <div class="row">
+        
+            <div class="col-md-6">
+                <%= f.label :NdaSigned, 'NDA Signed' do %>
+                    <%= 'NDA Signed' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :NdaSigned, :checked => false, :required => true %>
+                    <%= f.file_field :ndaDoc%>
+                <% end %>
+            </div>
+            
+            <div class="col-md-6">
+                <%= f.label :workstationReady, 'Workstation prepared for new employee?' do %>
+                    <%= 'Workstation prepared for new employee? ' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :workstationReady, :checked => false, :required => true %>
+                <% end %>
+            </div>
+    
+            <div class="col-md-6">
+                <%= f.label :packReady, 'Greeting pack prepared for new employee?' do %>
+                    <%= 'Greeting pack prepared for new employee?' %>
+                    <span class='required-asterisk'>*</span>
+                    <%= f.check_box :packReady, :checked => false, :required => true %>
```

Comment:
Change hash syntax

Reply:
changed apu
please check it once again just to make sure whether I've used proper syntax or not